pylmod/tests/test_gradebook.py

```diff
@@ -17,6 +18,7 @@
 from pylmod.tests.common import BaseTest
 
 
+@ddt
```

Comment:
Also, we do use pytest for this test suite but the pytest parametrize decorator does not work with `unittest.TestCase` classes